### DISCUSSION TITLE
security onion setup -2.3.110

### DISCUSSION THREAD

**Discussion by RihemJendoubi** [Security-Onion-Solutions/securityonion#10365] [General]
can anyone provide me with the iso image of security onion 2.3.110, i need this exact image

**Comment by cm-ops** [comment]
Would recommend the latest release obviously, but if you needed that image you could download here https://download.securityonion.net/file/securityonion/securityonion-2.3.110-20220407.iso

**Comment by RihemJendoubi** [comment]
can you help me with installing security onion with standalone deployment
mode, i tried it with import mode but it wasn't generating any alert after
importing a pcap file

Le mer. 17 mai 2023, 13:58, Chris Morgret ***@***.***> a
écrit :

**Comment by cm-ops** [comment]
Recommend you check out the free YouTube Essentials videos - https://www.youtube.com/watch?v=U--CXHmBDXQ&list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH and also the RTDs here: https://docs.securityonion.net/en/latest/getting-started.html
